paddle/fluid/operators/detail/send_recv.proto

```diff
@@ -69,6 +69,10 @@ message VariableMessage {
   bytes rows = 9;
   // Look up table block execution output variable name.
   string out_varname = 10;
+  // If true, the ps server will start profiling, the ps
+  // server stops profiling and generates a profile to /tmp/profile_ps_*
+  // when profile switches from true to false.
+  bool profile = 11;
```

Comment:
Can we just use a envronment variable to determine whether to run profile?

Reply:
Using env is difficult to start and stop profiling of multiple machines at the same time.